locksmith/__tests__/controllers/v2/lockSettingController.test.ts

```diff
@@ -52,14 +52,14 @@ vi.mock('@unlock-protocol/unlock-js', () => {
 })
 
 describe('LockSettings v2 endpoints for lock', () => {
-  it('should fail to get settings  with no authentication', async () => {
```

Comment:
Please add tests to make sure when the call is authenticated it includes the right data and when not authenticated it does not include it!